Sources/_SubprocessCShims/process_shims.c

```diff
@@ -692,13 +679,22 @@ int _subprocess_fork_exec(
         rc = closefrom(pipefd[1] + 1);
         #endif
         if (rc != 0) {
-            // close_range failed (or doesn't exist), fall back to close()
-            for (int fd = STDERR_FILENO + 1; fd <= _highest_possibly_open_fd(); fd++) {
+            #if defined(__linux__)
```

Comment:
Annoying GitHub won't let me comment on the right line but on line 671-672, for consistency with the other changes you just made, you'll want to pass `CLOSE_RANGE_CLOEXEC` to the two close_range calls above on Linux/FreeBSD.

OpenBSD doesn't have any efficient way to set CLOEXEC on a range of fds, so closefrom will have to do.